src/hooks/__tests__/useLogout.test.ts

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { useLogout } from "../useLogout";
+
+describe("useLogout", () => {
+  test.todo("should navigate to login page if success logout");
```

Comment:
I can only approve the PR if the tests actually work

Reply:
done